CHANGES.rst

```diff
@@ -1,6 +1,11 @@
 Changes
 =======
 
+Version 0.8.1 (UNRELEASED)
+---------------------------
+
+- Uses polling job-controller to obtain job statuses instead of checking files.
```

Comment:
What do you propose in this case? Some ideas:

1. two lines like:

```
- Removes checking files to determine job status
- Adds polling job-controller to determine job statuses
```

2. one like:

```
- Adds polling job-controller to determine job statuses instead of checking files
```

Reply:
I went with the second point to have related changes in one line.